README.md

```diff
@@ -138,7 +186,9 @@ module.exports = {
   plugins: [
     new MiniCssExtractPlugin({
       filename: "[name].css",
-      chunkFilename: "[id].css"
+      chunkFilename: "[id].css",
+      // disables order warnings being logged in terminal after build
+      orderWarning: true, // Disable to remove warnings about conflicting order between imports
```

Comment:
Let's avoid this option here, let's do it in other PR and add documentation how remove this using `warningsFilter`